### PR TITLE
More Webpack > v2 support changes

### PR DESCRIPTION
**Pull request by twhitbeck**

### CODE REVIEW

File: lib/i18nDictionary.js [hunk 1]
```diff
@@ -86,7 +86,9 @@ class Dictionary {
 			/** @type {Object.<string, JSONConcatSource>} */
 			let locales = {};
 			for ( const module of Array.from( chunk.modulesIterable ) ) {
-				locales = module.extractLocales( locales, this.shared_text_key )
+				if (module instanceof I18nModule) {
```
Comment (twhitbeck) [inline lib/i18nDictionary.js]: tbh, I don't know why this is necessary, but it didn't work without it.

Comment (nerdkid93) [reply lib/i18nDictionary.js]: previously, the only modules our `main` and `vendor` chunks had we're `i18nModules`. Now there are more modules in our chunks

Comment (nerdkid93) [inline lib/i18nDictionary.js]:
```suggestion
				if ( module instanceof I18nModule ) {
```

File: lib/i18nDictionary.js [hunk 2]
```diff
@@ -86,7 +86,9 @@ class Dictionary {
...
+					locales = module.extractLocales(locales, this.shared_text_key)
```
Comment (nerdkid93) [inline lib/i18nDictionary.js]:
```suggestion
					locales = module.extractLocales( locales, this.shared_text_key );
```

**Pull request merged**